roles/dns/manage-dns-zones-route53/README.md

```diff
@@ -1,10 +1,7 @@
 manage-dns-zones-route53
 ========================
 
-This role can be used to manage views and zones for different DNS service providers. The supported ones are:
-
-  - named
-  - route53
+This role can be used to manage views and zones for AWS Route 53.
```

Comment:
Can you please update this line as Route53 doesn't really have views - i.e.: it's just 2 fixed ones (public and private). Maybe change it to something like: 
> This role can be used to manage DNS zones within the two existing AWS Route53 views "public" and "private".